src/vs/base/parts/quickinput/common/quickInput.ts

```diff
@@ -146,16 +159,20 @@ export interface IQuickInput {
 	show(): void;
 
 	hide(): void;
-
-	onDidHide: Event<void>;
-
-	dispose(): void;
 }
 
 export interface IQuickPick<T extends IQuickPickItem> extends IQuickInput {
 
 	value: string;
 
+	/**
+	 * A method that allows to massage the value used
+	 * for filtering, e.g, to remove certain parts.
+	 */
+	filterValue: (value: string) => string;
+
+	ariaLabel: string;
```

Comment:
@chrmarti fyi this is to be able to set an ARIA label on the quick picker input field from the outside